runtime/src/snapshot_utils.rs

```diff
@@ -1459,6 +1858,40 @@ pub(crate) fn parse_incremental_snapshot_archive_filename(
     })
 }
 
+pub(crate) fn parse_appendvec_filename(filename: &str) -> Option<(Slot, AppendVecId)> {
```

Comment:
I think I had a funciton that did something similar, can we just change that to use regex? or replace its use with this one

Reply:
Done.